docs/src/main/paradox/scala/http/client-side/client-https-support.md

```diff
@@ -1,15 +1,19 @@
-<a id="clientsidehttps"></a>
 # Client-Side HTTPS Support
 
-Akka HTTP supports TLS encryption on the client-side as well as on the @ref[server-side](../server-side-https-support.md#serversidehttps-scala).
+Akka HTTP supports TLS encryption on the client-side as well as on the @ref[server-side](../server-side-https-support.md).
 
 The central vehicle for configuring encryption is the `HttpsConnectionContext`, which can be created using
 the static method `ConnectionContext.https` which is defined like this:
 
-@@snip [ConnectionContext.scala](../../../../../../../akka-http-core/src/main/scala/akka/http/scaladsl/ConnectionContext.scala) { #https-context-creation }
+Scala
+:  @@snip[ConnectionContext.scala](../../../../../../../akka-http-core/src/main/scala/akka/http/scaladsl/ConnectionContext.scala) { #https-context-creation }
+
+Java
+:  @@snip [ConnectionContext.scala](../../../../../../../akka-http-core/src/main/scala/akka/http/javadsl/ConnectionContext.scala) { #https-context-creation }
```

Comment:
I see that the previous version also pointed to a Scala file, but shall we do something here to not show Scala code on the Java documentation?